Bottom: 3c80ce2a67baeab96fd06c350716dccc4bb7ad72
Top:    0cf18140759f544d7939aa62d2647161675c47b4
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 14:44:33 +0800

Refresh of autofs-5.0.7-document-browse-option-in-man-page.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 50966bc..df40b41 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -35,6 +35,7 @@
 - fix requires in spec file.
 - fix libtirpc build option to require libtirpc-devel if needed.
 - fix systemd unidir in spec file.
+- document browse option in man page.
 
 25/07/2012 autofs-5.0.7
 =======================
